tests/src/common/Wsk.scala

```diff
@@ -252,13 +252,13 @@ class WskAction()
      */
     def invoke(
         name: String,
-        parameters: Map[String, JsValue] = Map(),
+        parameters: Map[String, String] = Map(),
```

Comment:
> It appeared though strings are being converted to JsValues, then converted back to strings right before the commands are executed. Is there a reason that is?
> 

The type enforces the contract that the cli supports values for parameters which are valid json values. The interface avoids and discourages trying to write Json as string and instead going through a proposed serializer.